### PR TITLE
Add a wall

### PR DESCRIPTION
**Pull request by akshit15013**
Description : Add a wall redright
Preview link : https://pr-823-fossasia-labyrinth.surge.sh/
screenshot 129
screenshot 131

### CODE REVIEW

**Comment by akshit15013** [comment]
I do not know how to do that.

**Comment by akshit15013** [comment]
Could you please merge it?

**Comment by YashKumarVerma** [comment]
You can learn from this pr : https://github.com/fossasia/labyrinth/pull/550

**Comment by akshit15013** [comment]
I am not able to get that.

**Comment by akshit15013** [comment]
Could you please help me?

**Comment by YashKumarVerma** [comment]
yea. sure.

Can you post the errors you are facing in the gitter channel ?

**Comment by akshit15013** [comment]
I have posted there also

**Comment by akshit15013** [comment]
I HAVE A MODIFIED CODE, CAN YOU PLEASE MERGE IT?

**Comment by akshit15013** [comment]
I HAVE A MODIFIED CODE, CAN YOU PLEASE MERGE IT?

File: js/tiles.js [hunk 1]
```diff
@@ -495,6 +495,20 @@ const door = {
             alertWarning("Be alert!!!", "You entered to the night!");         
         },
     }),
+    reddish: Object.assign({}, OpenDoors, {
+    	canEnterFromTheRight() {
```
Comment (STEdirisuriya) [inline js/tiles.js]: See here, you have disables enter and leave from the right side.

File: js/tiles.js [hunk 2]
```diff
@@ -495,6 +495,20 @@ const door = {
...
+	    return false;
+    },
+        canLeaveToTheRight() {
+            return false;
+    },
+    
+    createImages: function() {
+      this.wallTop = this.createImage("tiles/rooms/wall/redright.svg"); 
+      this.wallRight = this.createImage("tiles/rooms/door/right.svg");
```
Comment (STEdirisuriya) [inline js/tiles.js]: But here, you have added a door to the right side.

**Comment by akshit15013** [comment]
I HAVE A MODIFIED CODE, CAN YOU PLEASE MERGE IT?

**Comment by akshit15013** [comment]
I HAVE A MODIFIED CODE, CAN YOU PLEASE MERGE IT?

**Comment by STEdirisuriya** [comment]
Congrats :+1:

**Pull request merged**